src/browser/NativeMessageInstaller.cpp

```diff
@@ -209,7 +209,7 @@ QString NativeMessageInstaller::getNativeMessagePath(SupportedBrowsers browser)
     QString basePath;
 #if defined(Q_OS_WIN)
     // If portable settings file exists save the JSON scripts to the application folder
-    if (QFile::exists(QCoreApplication::applicationDirPath() + QStringLiteral("/keepassxc.ini"))) {
+    if (QFile::exists(QCoreApplication::applicationDirPath() + QStringLiteral("/.portable"))) {
```

Comment:
May be best to move this to an `isPortable()` function, especially if that check is done in more than one place.
Also, the comment still says "portable settings file". Should probably say "portable marker", or just "In portable mode".

Reply:
Turn this into a function of Config: https://github.com/keepassxreboot/keepassxc/blob/develop/src%2Fcore%2FConfig.cpp#L522

Then use it there and here.

Reply:
would do that, but for that I am not versed enough in cpp to safely and securely code it. I don't even a way to compile keepassxc (or any cpp project), so testing is also out of the window.
Throw me a base C-Project (or Java, or PHP) and I could do more, but for cpp I am unfortunately only able to do small fixes like this, which don't really change the structure of the code

Reply:
I'd like to point out that changing the behaviour of this code may result in unexpected failures for people who rely on it.

Reply:
> would do that, but for that I am not versed enough in cpp to safely and securely code it. I don't even a way to compile keepassxc (or any cpp project), so testing is also out of the window. Throw me a base C-Project (or Java, or PHP) and I could do more, but for cpp I am unfortunately only able to do small fixes like this, which don't really change the structure of the code

I attempted it in a local repository, but:
- Config: `portablePath` would need to become a member-attribute or method `getDefaultConfigs()` would need an additional parameter, or we would duplicate code, leaving us open for something similar to happen in the future
- Config: alternatively: set `m_isPortable` (from `isPortable`) inside `getDefaultConfigs()`, but having to call the function in the parameterized Constructor aswell, even tough it serves no other purpose
- Config: can't put it into `init`, as it is called after `getDefaultConfigs()`, which requires said configs.

I am absolutely not comfortable in making that change myself.